python/adelphi/README.md

```diff
@@ -0,0 +1,107 @@
+# Adelphi tooling
+
```

Comment:
What do you think about adding a quick intro block to explain the high level of the script and what all it can do?  That will give a bit more context to the detailed commands you cover below, I think.

Reply:
Maybe it can include some info on how to install the package via pip and what the basic system requirements are (python version, anything else)?

Reply:
Yeah, I think that makes sense @jdonenine.  It seemed weird to add that kind of thing to a doc that's appearing on PyPI where presumably users are either already aware of those details of they're specified via classifiers and other metadata in setup.py.  But the README here is serving two distinct roles; it's providing content for PyPI _and_ it's serving as a general README in the Github repo.  The block you're describing makes a ton of sense for the second role so I'll add that in there.

Reply:
Agree on people getting here likely by knowing what they were looking for, so not a lot of detail necessary I'd think.

Reply:
I tried to address this in the "Introduction" section.  Take a look and see if you're satisfied with what's there @jdonenine

Reply:
I like it, thanks for adding.